mobile/lib/screens/dashboard/dashboard_view.dart

```diff
@@ -52,17 +53,56 @@ class _DashboardViewState extends State<DashboardView>
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(
-        automaticallyImplyLeading: false,
-        centerTitle: false,
-        title: SvgPicture.asset(
-          'assets/icon/airqo_logo.svg',
-          height: 40,
-          width: 58,
-          semanticsLabel: 'AirQo',
+      appBar: PreferredSize(
+        preferredSize: const Size.fromHeight(60.0),
+        child: Showcase.withWidget(
+          key: _skipShowcaseKey,
+          overlayOpacity: 0.6,
+          shapeBorder: const RoundedRectangleBorder(),
+          width: 50,
+          height: 50,
+          container: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: <Widget>[
+              Container(
+                width: 45,
+                height: 45,
+                decoration: const BoxDecoration(
+                  shape: BoxShape.circle,
+                  color: Colors.white,
+                ),
+                child: IconButton(
+                  tooltip: "Skip Showcase",
+                  icon: const Icon(Icons.skip_next),
+                  onPressed: () => {
+                    ShowCaseWidget.of(context).dismiss(),
+                  },
```

Comment:
Hi @BenjaminSsempala I thin this can be simplified to
```dart
onPressed: () => ShowCaseWidget.of(context).dismiss(),
```

Reply:
Thanks @NoahNsimbe . I've made the changes.